packages/frontend-2/components/project/page/InviteDialog.vue

```diff
@@ -48,34 +51,37 @@
         />
       </div>
     </div>
-  </LayoutDialogSection>
+  </LayoutDialog>
 </template>
 <script setup lang="ts">
 import { Roles } from '@speckle/shared'
 import type { ServerRoles, StreamRoles } from '@speckle/shared'
-import { LayoutDialogSection } from '@speckle/ui-components'
 import { useUserSearch } from '~~/lib/common/composables/users'
 import type { UserSearchItem } from '~~/lib/common/composables/users'
 import type {
   ProjectInviteCreateInput,
-  ProjectPageTeamDialogFragment
+  ProjectPageSettingsCollaboratorsQuery
 } from '~~/lib/common/generated/gql/graphql'
 import type { SetFullyRequired } from '~~/lib/common/helpers/type'
 import { isEmail } from '~~/lib/common/helpers/validation'
 import { isArray, isString } from 'lodash-es'
 import { useInviteUserToProject } from '~~/lib/projects/composables/projectManagement'
-import { useTeamDialogInternals } from '~~/lib/projects/composables/team'
-import { UserPlusIcon } from '@heroicons/vue/24/outline'
+import { useTeamInternals } from '~~/lib/projects/composables/team'
 import { useMixpanel } from '~~/lib/core/composables/mp'
 import { useServerInfo } from '~~/lib/core/composables/server'
 
 type InvitableUser = UserSearchItem | string
 
 const props = defineProps<{
-  project: ProjectPageTeamDialogFragment
-  defaultOpen: boolean
+  project?: ProjectPageSettingsCollaboratorsQuery
+  disabled?: boolean
+  disabledMessage?: string
```

Comment:
What's the point of this prop, if the message never gets used? We do check for the existence of this prop in the template, but then it's not clear why this exists and it probably has to be renamed to something more intuitive.

From the name I would think that this allows feeding in a custom disabled message